packages/api/src/sites/sites.entity.ts

```diff
@@ -167,6 +167,8 @@ export class Site {
 
   maskedSpotterApiToken?: string;
 
+  waterQuality?: string[];
```

Comment:
I suggest we rename this field to indicate this is a list of strings(Sources), maybe:

```suggestion
  waterQualitySources?: string[];
```